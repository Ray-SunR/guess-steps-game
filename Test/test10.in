game 3 2 test10A1.txt test10A1.txt
quit
