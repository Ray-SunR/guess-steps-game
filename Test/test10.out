Total is 3, max deduction is 2
Player A's move
Total is 2
Player B's move
Total is 1
Player A's move
Total is 0
Player A wins
Score is
A 1
B 0
